src/components/preconfigured/Standard.tsx

```diff
@@ -8,7 +8,10 @@ export const Standard = (props: WidgetProps) => {
   const params = new URLSearchParams(props.location.search)
   const demo = params.has('demo')
   const backgroundImage = params.get('withoutBackgroundImage') === 'true' ? null : defaultBackgroundImage
-  const textColor = params.get('textColor') || 'fff'
+  let textColor = 'ffffff'
+  if (params.get('withoutBackgroundImage') === 'true') {
+    textColor = params.get('textColor') || 'ffffff'
+  }
```

Comment:
For consistency with the ternary evaluation above I would do this as:
```suggestion
  const textColor = backgroundImage && params.get('textColor') ? params.get('textColor') : 'ffffff'
```

Reply:
I leave it as is. Decision was approved by Simon.

Reply:
It’s also fine to leave it as is. (As we just found out that TS would require the expression to be a bit more complex, decreasing its readability.)

Reply:
```suggestion
  const textColor = backgroundImage && params.get('textColor') || 'ffffff'
```
This should have worked 🤦